qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -86,6 +86,9 @@
             no Windows
             gagent_check_type = fsfreeze
             gagent_fs_test_cmd = "rm -f /tmp/foo; echo foo > /tmp/foo"
+        - check_reset_reboot_shutdown_fsfreeze:
+            no Windows
```

Comment:
Checked polarion case https://polarion.engineering.redhat.com/polarion/#/project/RedHatEnterpriseLinux7/workitem?id=RHEL7-8350

This case is linux only?